examples/step-70/step-70.cc

```diff
@@ -1749,7 +1749,7 @@ namespace Step70
     static std::map<std::string, std::vector<std::pair<double, std::string>>>
       times_and_names;
     if (times_and_names.find(fprefix) != times_and_names.end())
-      times_and_names[fprefix].push_back(std::make_pair(time, filename));
+      times_and_names[fprefix].emplace_back(time, filename);
```

Comment:
I find this hard to read and have resisted doing so in the tutorials. The `times_and_names` is an array of pairs. The syntax used here isn't clear that that's what we're storing. Would you mind changing to the following?
```suggestion
      times_and_names[fprefix].emplace_back({time, filename});
```

Reply:
That kind of defeats the purpose of this change which is to not call the constructor directly instead of constructing the object in place. I find the purpose here pretty clear (we have a variable called `times_and_names` and store a time and a name) but I'm OK reverting this if you feel strongly about it.

Reply:
I'll let others chime in. It calls the constructor with an initializer list; it doesn't actually construct an object in place, if I understand things right.

But I don't feel particularly adamant about it.

Reply:
I like the `emplace_back()` version: like Daniel said, it forwards the arguments to `emplace_back()` to the correct constructor of the value type to do in-place construction in the 'normal' way (no `std::initializer_list`).

Reply:
I also vote for @masterleinad's `emplace_back`.